### PR TITLE
[HTD-27] Feat: Add `Blog CTA block` Component

### PR DESCRIPTION
**Pull request by RBoraurito**
## Assets

| Light | Dark |
| ------ | ----- |
|    image    |   image    |

## Before Merge Checklist

The following steps were checked before merging a PR

- [x] checked and fixed code style issues
- [x] checked and removed unnecessary/junky code
- [x] manually tested introduced solution
- [ ] added needed tests to automate testing this code in the future

### CODE REVIEW

File: src/components/modules/blog/cta/index.tsx [hunk 1]
```diff
@@ -0,0 +1,20 @@
+import { StyledBlogCTASubtitle, StyledBlogCTAWrapper } from './cta.styles';
+import type { BlogCTAProps } from './cta.types';
+
+import { Button } from '@/components/elements/button';
+import { Heading } from '@/components/elements/heading';
+import { CaretRight } from '@/components/elements/icon';
+
+export const BlogCTA = ({ title, buttonText, subtitle, buttonHref }: BlogCTAProps) => {
```
Comment (nephlin7) [inline src/components/modules/blog/cta/index.tsx]: @RBoraurito, There is a similar box in the [FAQ](https://www.figma.com/file/O5411bxKHP2O2OyfW19w6E/HTTP-Toolkit---UI-Design?type=design&node-id=410-14582&mode=design&t=HvFdqTpp01XaWWod-0) pages, Could we move and name this to a more generic name to reuse it in the two places? Something like BoxCTA or similar:

Screenshot 2024-02-28 at 19 49 12

File: src/components/modules/blog/cta/index.tsx [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+  return (
+    <StyledBlogCTAWrapper>
+      <StyledBlogCTASubtitle>{subtitle}</StyledBlogCTASubtitle>
+      <Heading fontSize="m" color="textGradient">
```
Comment (nephlin7) [inline src/components/modules/blog/cta/index.tsx]: @RBoraurito, the default heading tag is H1. Shouldn't this be an h3 or something?

Comment (RBoraurito) [reply src/components/modules/blog/cta/index.tsx]: Think the same, it should be an H3

File: src/components/modules/blog/cta/index.tsx (hunk 1 above)
Comment (RBoraurito) [reply src/components/modules/blog/cta/index.tsx]: image
image

Comment (RBoraurito) [reply src/components/modules/blog/cta/index.tsx]: I just add the icon to the blog cta example

**Review by nephlin7 — APPROVED** [review]

**Comment by nephlin7** [comment]
Great job @RBoraurito 💪🏽

**Pull request merged**